README.txt
                    WordPress packaged by Bitnami 5.9.3-1
                  ==============================


1. OVERVIEW

The Bitnami Project was created to help spread the adoption of freely available, 
high quality, open source web applications. Bitnami aims to make it easier than 
ever to discover, download and install open source software such as document and 
content management systems, wikis and blogging software.

You can learn more about Bitnami at https://bitnami.com

WordPress is the world's most popular blogging and content management platform. 
Powerful yet simple, everyone from students to global corporations use it to 
build beautiful, functional websites.

You can learn more about WordPress at http://www.wordpress.org

The WordPress packaged by Bitnami is an installer that greatly simplifies the
installation of WordPress and runtime dependencies. It includes ready-to-run
versions of Apache, MariaDB and PHP.

WordPress packaged by Bitnami is distributed for free under the Apache 2.0 
license. Please see the appendix for the specific licenses of all open source 
components included.

You can learn more about Bitnami Stacks at https://bitnami.com/stacks/

2. FEATURES

- Easy to Install

Bitnami Stacks are built with one goal in mind: to make it as easy as
possible to install software. Our installers completely automate
the process of installing and configuring all of the software included in
each Stack, so you can have everything up and running in just a few clicks.

- Independent

Bitnami Stacks are completely self-contained, and therefore do not interfere
with any software already installed on your system. For example, you can
upgrade your system's MariaDB or Apache without fear of 'breaking' your
Bitnami Stack.

- Integrated

By the time you click the 'finish' button on the installer, the whole stack
will be integrated, configured and ready to go.

- Relocatable

Bitnami Stacks can be installed in any directory. This allows you to have
multiple instances of the same stack, without them interfering with each other.

3. COMPONENTS

WordPress packaged by Bitnami ships with the following software versions:

  - WordPress 5.9.3
  - WP-CLI 2.6.0
  - Apache 2.4.53
  - Varnish(TM) (Only supported on Linux and OS X)
  - MariaDB 10.4.24
  - PHP 8.0.19
  - phpMyAdmin 5.2.0

4. REQUIREMENTS

To install WordPress packaged by Bitnami you will need:

    - Intel x86 or compatible processor
    - Minimum of 700 MB RAM
    - Minimum of 150 MB hard drive space
    - TCP/IP protocol support
    - Compatible operating systems:
      - An x86 or x64 Linux operating system.
      - A Windows operating system such as Windows Vista, Windows 7, Windows 8, Windows 10, Windows Server 2008 or Windows Server 2012.
			- An OS X operating system.

5. INSTALLATION

The WordPress packaged by Bitnami is distributed as a binary executable installer.
It can be downloaded from:

https://bitnami.com/stacks/

The downloaded file will be named something similar to:

bitnami-wordpress-5.9.3-1-linux-x64-installer.run on Linux 64 bit or
bitnami-wordpress-5.9.3-1-windows-installer.exe on Windows or
bitnami-wordpress-5.9.3-1-osx-x64-installer.dmg on OS X.

On Linux, you will need to give it executable permissions:

chmod 755 bitnami-wordpress-5.9.3-1-linux-x64-installer.run


To begin the installation process, double-click on that file, and you will
be greeted by the 'Welcome' screen. Pressing 'Next' will take you to the
Component Selection screen, where you can select the phpMyAdmin component. This
tool intended to handle the administration of MariaDB over the web.

The next screen is the Installation Folder, where you can select where Bitnami
stack will be installed. If the destination directory does not exist, it will
be created as part of the installation.

The next screen will prompt you for data necessary to create the initial
admin user:

Username and password: You will use this information to log-in into the
administrative interface. The password you provide here will also be used to
protect other parts of the installation. Please see the section named
"Usernames and Passwords" later in this document.

Real name: The name that will be displayed in the application.

Email address: Your email address.

The next screen will vary, depending on whether the ports needed by the
bundled Apache and MariaDB are already taken. The default listening port for
Apache is 8080 on Linux or port 80 on Windows and for MariaDB is 3006. If those
ports are already in use by other applications, you will be prompted for
alternate ports to use.

The next screen will allow you to configure the final details of your
WordPress installation:

Blog name: Title of your blog, which will be displayed to visitors

Hostname: The hostname for your blog, such as www.example.com. This
information will be used by WordPress when creating certain links.

Once the information has been entered, the installation will proceed to copy
the files to the target installation directory and configure the different
components. One this process has  been completed, you will see the
'Installation Finished' page. You can choose to launch WordPress packaged by Bitnami
at this point. If you do so, your default web browser will open and
display the Welcome page for the WordPress packaged by Bitnami.

If you received an error message during installation, please refer to
the Troubleshooting section.

The rest of this guide assumes that you installed Bitnami WordPress Stack
in /home/user/wordpress-5.9.3-1 on Linux or
C:\Bitnami\wordpress-5.9.3-1 on Windows or
/Applications/wordpress-5.9.3-1 on OS X and that you use port 8080
for Apache on Linux or port 80 on Windows and 3306 for MariaDB.

6. STARTING AND STOPPING WORDPRESS PACKAGED BY BITNAMI

The Bitnami Application Manager is a simple graphical interface included in
the stacks that can start and stop the Bitnami servers. It is are located
in the same installation directory.

To start the utility, double click the file named ‘manager-linux’,
'manager-windows' or 'manager-osx' from your file browser.

Alternatively, you can also start and stop the services manually, as explained below.

To start/stop/restart application on Linux and OSX you can use the included ctlscript.sh
utility, as shown below:

       ./ctlscript.sh (start|stop|restart)
       ./ctlscript.sh (start|stop|restart) mariadb
       ./ctlscript.sh (start|stop|restart) apache
       ./ctlscript.sh (start|stop|restart) varnish

  start      - start the service(s)
  stop       - stop  the service(s)
  restart    - restart or start the service(s)

On Windows, you can start and stop WordPress packaged by Bitnami using the shortcuts created
in the Start Menu, under Programs -> Bitnami -> Bitnami Service

That will start Apache service. Once started, you can open your
browser and access the following URL on Linux or OS X:

http://127.0.0.1:8080/wordpress/

And you can access the following URL on Windows:

http://127.0.0.1/wordpress/

If you selected an alternate port during installation, for example 18080, the
URL will look like:

http://127.0.0.1:18080/wordpress/

If you are accessing the machine remotely, you will need to replace
127.0.0.1 with the appropriate IP address or hostname.

7. DIRECTORY STRUCTURE

The installation process will create several subfolders under the main
installation directory:

	apache2/: Apache Web server.
        varnish/: Varnish(TM) Cache
	php/: PHP Scripting Language.
	mariadb/: MariaDB Database.
	apps/
	  wordpress/: WordPress application folder
	    conf/: WordPress Apache configuration files
	    htdocs/: WordPress PHP application files
	  phpMyAdmin/: phpMyAdmin application folder (optional)

8. DEFAULT USERNAMES AND PASSWORDS

The WordPress administrative user and password are the ones you set at
installation time.

MariaDB admin user is called 'root', and its password is the same as the
WordPress administrative user password.

The default MariaDB non-root account used to access the bitnami_wordpress database
is named bn_wordpress, and its password is random.

The phpMyAdmin administrative user is 'root'.

9. TROUBLESHOOTING

You can find more information about this product at:

https://docs.bitnami.com/

We also encourage you to post your questions and suggestions at:

https://community.bitnami.com

We also encourage you to sign up for our newsletter, which we'll use to
announce new releases and new stacks. To do so, just register at:
https://bitnami.com/newsletter.

9.1 Installer

# Installer Payload Error

If you get the following error while trying to run the installer from the
command line:

"Installer payload initialization failed. This is likely due to an
incomplete or corrupt downloaded file"

The installer binary is not complete, likely because the file was
not downloaded correctly. You will need to download the file and
repeat the installation process.

9.2 Apache

If you find any problem starting Apache, the first place you should check is
the Apache error log file:

/home/user/wordpress-5.9.3-1/apache2/logs/error.log on Linux or
C:\bitnami\wordpress-5.9.3-1\apache2\logs\error.log on Windows or
/Applications/wordpress-5.9.3-1/apache2/logs/error.log.

Most errors are related to not being able to listen to the default port.
Make sure there are no other server programs listening at the same port
before trying to start Apache. Some programs, such as Skype, may also use
port 80. For issues not covered in this Quick Start guide, please refer to
the Apache documentation, which is located at http://httpd.apache.org/docs/

9.3 MariaDB

The following are some common problems:

# Access denied when trying to connect to MariaDB.

If you get an Access Denied message while trying to connect to
MariaDB, make sure you are using the correct username and password.

# "Can't connect to server" message.

Make sure that the MariaDB daemon is up and running. Remember also that if
during installation you selected a different listening port for MariaDB, you
may need to pass that as an extra command line option.

For help in troubleshooting MariaDB errors, you may want to reference the
the MariaDB manual, which you will find at https://mariadb.com/kb/en/documentation/

9.4 Varnish(TM)

# Changing Varnish(TM) port

The installer will automatically set for the Varnish(TM) port the first one free after
the Apache port. This works fine to test it but if you want to use Varnish(TM) in a
production environment, you will  probably want to change its port to a more
appropriate one (for example, 80 instead of 81) and change Apache port if it is
currently taking it.

You can find a guide of how to do this in the below link:

https://docs.bitnami.com/general/apps/wordpress/administration/configure-use-varnish/

# Configuring Varnish(TM) to use a WordPress-specific configuration file

By default, Varnish(TM) uses the default configuration settings. To make WordPress
properly handle when the cache must be refreshed or ignored, that configuration
file must be replaced. The below article explains how to configure Varnish(TM)
with WordPress:

https://docs.bitnami.com/general/apps/wordpress/administration/configure-use-varnish/

9.5 WordPress

For any problem related to WordPress, please visit
http://codex.wordpress.org/Troubleshooting


10. LICENSES

Apache Web Server is distributed under the Apache License v2.0, which
is located at http://www.apache.org/licenses/LICENSE-2.0

Varnish(TM) cache is distributed under the BSD License, located at
http://www.opensource.org/licenses/bsd-license.php

MariaDB is distributed under the GNU General Public License v2, which is
located at http://www.gnu.org/licenses/old-licenses/gpl-2.0.html

PHP and related libraries are distributed under the PHP License v3.01,
which is located at http://www.php.net/license/3_01.txt

WordPress is distributed under the GNU General Public License v2,
which is located at
http://www.gnu.org/licenses/old-licenses/gpl-2.0.html

curl is distributed under the Curl License, which is located at
http://curl.haxx.se/docs/copyright.html

expat is distributed under the MIT License, which is located at
http://www.jclark.com/xml/copying.txt

gd is distributed under the gd License, which is located at
http://www.boutell.com/gd/manual2.0.33.html

IMAP is distributed under the University of Washington Free-Fork
License, located at http://www.washington.edu/imap/legal.html

jpegsrc is distributed under The Independent JPEG Group's JPEG
software license, which is located at
http://dev.w3.org/cvsweb/Amaya/libjpeg/README?rev=1.2

libiconv is distributed under the Lesser General Public License
(LGPL), located at http://www.gnu.org/copyleft/lesser.html

Freetype is released under The Freetype Project License, that is located at
http://freetype.sourceforge.net/FTL.TXT


changelog.txt
============ CHANGELOG =============

Version 5.9.3-1      2022-05-17
* Updated OpenSSL to 1.1.1o
* Updated PHP to 8.0.19
* Updated phpMyAdmin to 5.2.0
* Updated SQLite to 3.38.5

Version 5.9.3-0      2022-04-06
* Updated Apache to 2.4.53
* Updated MariaDB to 10.4.24
* Updated OpenSSL to 1.1.1n
* Updated PHP to 8.0.17
* Updated SQLite to 3.38.2
* Updated WordPress to 5.9.3

Version 5.9.2-0     2022-03-15
* Updated PHP to 7.4.28
* Updated SQLite to 3.38.0
* Updated WordPress to 5.9.2

Version 5.9.1-0      2022-03-02
* Updated phpMyAdmin to 5.1.3
* Updated WordPress to 5.9.1

Version 5.9-0      2022-02-08
* Updated lego to 4.6.0
* Updated phpMyAdmin to 5.1.2
* Updated SQLite to 3.37.2
* Updated Varnish to 6.0.10
* Updated WordPress to 5.9
* Updated WP-CLI to 2.6.0

Version 5.8.3-0      2022-01-07
* Updated WordPress to 5.8.3

Version 5.8.2-2      2021-12-25
* Maintenance release
* Updated Apache to 2.4.52
* Updated MariaDB to 10.4.22
* Updated OpenSSL to 1.1.1m
* Updated PHP to 7.4.27
* Updated SQLite to 3.37.0
* Updated Varnish to 6.0.9

Version 5.8.2-0      2021-11-12
* Updated PHP to 7.4.25
* Updated WordPress to 5.8.2

Version 5.8.1-2      2021-10-08
* Fix Apache Zero Day vulnerability \(CVE-2021-42013\)
* Updated Apache to 2.4.51

Version 5.8.1-1      2021-10-07
* Fixed Apache Zero Day vulnerability (CVE-2021-41773)
* Updated Apache to 2.4.50
* Updated lego to 4.5.3
* Updated PHP to 7.4.24

Version 5.8.1-0      2021-09-10
* Updated WordPress to 5.8.1

Version 5.8-1      2021-09-09
* Maintenance release
* Updated MariaDB to 10.4.21
* Updated OpenSSL to 1.1.1l
* Updated PHP to 7.4.23

Version 5.8-0      2021-07-28
* Added Imagemagick PHP extension for Windows
* Updated Apache to 2.4.48
* Updated lego to 4.4.0
* Updated MariaDB to 10.4.20
* Updated PHP to 7.4.21
* Updated phpMyAdmin to 5.1.1
* Updated SQLite to 3.36.0
* Updated Varnish to 6.0.8
* Updated WordPress to 5.8
* Updated WP-CLI to 2.5.0

Version 5.7.2-0      2021-05-13
* Updated MariaDB to 10.4.19
* Updated PHP to 7.4.19
* Updated WordPress to 5.7.2

Version 5.7.1-2      2021-05-06
* Migrated DB to MariaDB
* Updated Apache to 2.4.47
* Updated MariaDB to 10.4.18
* Updated PHP to 7.4.18

Version 5.7.1-1      2021-05-01
* Update hyperkit binary for the OS X VM solution
* Updated MySQL to 8.0.24
* Updated SQLite to 3.35.5

Version 5.7.1-0      2021-04-15
* Updated SQLite to 3.35.4
* Updated WordPress to 5.7.1

Version 5.7-2      2021-03-26
* Fix wp-cli permissions
* Updated OpenSSL to 1.1.1k
* Updated SQLite to 3.35.2

Version 5.7-0      2021-03-19
* Updated lego to 4.3.1
* Updated PHP to 7.4.16
* Updated phpMyAdmin to 5.1.0
* Updated SQLite to 3.35.1
* Updated WordPress to 5.7

Version 5.6.2-0      2021-02-22
* Updated OpenSSL to 1.1.1j
* Updated WordPress to 5.6.2

Version 5.6.1-1      2021-02-15
* Maintenance release
* Updated MySQL to 8.0.23
* Updated PHP to 7.4.15

Version 5.6.1-0      2021-02-04
* Updated WordPress to 5.6.1

Version 5.6-3      2021-01-26
* Updated lego to 4.2.0
* Updated SQLite to 3.34.1

Version 5.6-2      2021-01-20
* Maintenance release
* Updated OpenSSL to 1.1.1i
* Updated PHP to 7.4.14
* Updated SQLite to 3.34.0

Version 5.6-0      2020-12-10
* Updated lego to 4.1.3
* Updated PHP to 7.4.13
* Updated WordPress to 5.6

Version 5.5.3-1      2020-11-17
* Updated lego to 4.1.0
* Updated PHP to 7.4.12
* Updated Varnish to 6.0.7

Version 5.5.3-0      2020-10-31
* Updated MySQL to 8.0.22
* Updated phpMyAdmin to 5.0.4
* Updated WordPress to 5.5.3

Version 5.5.2-0      2020-10-30
* Updated MySQL to 8.0.22
* Updated phpMyAdmin to 5.0.4
* Updated WordPress to 5.5.2

Version 5.5.1-2      2020-10-14
* Added Support for macOS Big Sur
* Updated OpenSSL to 1.1.1h
* Updated PHP to 7.4.11
* Updated phpMyAdmin to 5.0.3

Version 5.5.1-1      2020-09-24
* Updated lego to 4.0.1
* Updated PHP to 7.4.10

Version 5.5.1-0      2020-09-01
* Updated WordPress to 5.5.1

Version 5.5-1      2020-08-26
* Updated Apache to 2.4.46
* Updated PHP to 7.4.9
* Updated SQLite to 3.33.0

Version 5.5-0      2020-08-12
* Updated WordPress to 5.5

Version 5.4.2-2      2020-08-06
* Maintenance release
* Updated MySQL to 8.0.21
* Updated PHP to 7.4.8

Version 5.4.2-1      2020-07-07
* Updated lego to 3.8.0
* Updated PHP to 7.4.7
* Updated SQLite to 3.32.3

Version 5.4.2-0      2020-06-15
* Updated lego to 3.7.0
* Updated MySQL to 8.0.20
* Updated PHP to 7.3.18
* Updated SQLite to 3.32.2
* Updated WordPress to 5.4.2

Version 5.4.1-0      2020-04-30
* Updated lego to 3.6.0
* Updated PHP to 7.3.17
* Updated WordPress to 5.4.1

Version 5.4-1      2020-04-22
* Fixed security vulnerability in OpenSSL CVE-2020-1967
* Updated OpenSSL to 1.1.1g

Version 5.4-0      2020-04-10
* Updated Apache to 2.4.43
* Updated lego to 3.5.0
* Updated OpenSSL to 1.1.1f
* Updated PHP to 7.3.16
* Updated phpMyAdmin to 5.0.2
* Updated SQLite to 3.31.1
* Updated WordPress to 5.4

Version 5.3.2-3      2020-02-19
* Maintenance release
* Updated lego to 3.3.0
* Updated PHP to 7.3.14
* Updated SQLite to 3.31.1.
* Updated Varnish to 6.0.6
* Updated WP-CLI to 2.4.1

Version 5.3.2-2      2020-01-09
* Updated phpMyAdmin to 5.0.1

Version 5.3.2-1      2020-01-07
* Maintenance release
* Updated All in One SEO Pack to 3.3.4
* Updated All-in-One WP Migration to 7.14
* Updated Google Analytics plugin to 7.10.2
* Updated PHP to 7.3.13
* Updated phpMyAdmin to 5.0.0

Version 5.3.2-0      2019-12-19
* Updated All-in-One WP Migration to 7.13
* Updated WordPress to 5.3.2
* Updated WP Mail SMTP to 1.8.1

Version 5.3.1-0      2019-12-13
* Updated All in One SEO Pack to 3.3.3
* Updated All-in-One WP Migration to 7.12
* Updated Google Analytics plugin to 7.10.1
* Updated Jetpack to 8.0
* Updated PHP to 7.3.12
* Updated phpMyAdmin to 4.9.2
* Updated Simple Tags to 2.5.7
* Updated WordPress to 5.3.1
* Updated WP Mail SMTP to 1.8.0

Version 5.3-0      2019-11-14
* Updated Akismet Anti-Spam to 4.1.3
* Updated All in One SEO Pack to 3.3.2
* Updated All-in-One WP Migration to 7.10

apps/wordpress/htdocs/wp-config.php
<?php
/**
 * The base configuration for WordPress
 *
 * The wp-config.php creation script uses this file during the installation.
 * You don't have to use the web site, you can copy this file to "wp-config.php"
 * and fill in the values.
 *
 * This file contains the following configurations:
 *
 * * Database settings
 * * Secret keys
 * * Database table prefix
 * * ABSPATH
 *
 * @link https://wordpress.org/support/article/editing-wp-config-php/
 *
 * @package WordPress
 */

// ** Database settings - You can get this info from your web host ** //
/** The name of the database for WordPress */
define( 'DB_NAME', 'bitnami_wordpress' );

/** Database username */
define( 'DB_USER', 'bn_wordpress' );

/** Database password */
define( 'DB_PASSWORD', 'f1f34aa0b0' );

/** Database hostname */
define( 'DB_HOST', '127.0.0.1:3307' );

/** Database charset to use in creating database tables. */
define( 'DB_CHARSET', 'utf8' );

/** The database collate type. Don't change this if in doubt. */
define( 'DB_COLLATE', '' );

/**#@+
 * Authentication unique keys and salts.
 *
 * Change these to different unique phrases! You can generate these using
 * the {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org secret-key service}.
 *
 * You can change these at any point in time to invalidate all existing cookies.
 * This will force all users to have to log in again.
 *
 * @since 2.6.0
 */
define('AUTH_KEY',         '4cec9ecc6d9cbfb55bd15bc60c060c3861f02ed27934a9e02deb84f3a89b95cb');
define('SECURE_AUTH_KEY',  '6ee8f583f3f627b6642adcd66cfcdca1875262f17808aeb86c4410503c7caf66');
define('LOGGED_IN_KEY',    '0fdc2b676ec28d15349b49e39fa0862af1c098faa93c142e07cfe522429e8bae');
define('NONCE_KEY',        '12dfab071ddc43c76c892eeae3fe16e729ba2d9068b288e1fe22f1677d03a1e8');
define('AUTH_SALT',        '78d3873375ef60b51fab86fe2fe556dc8c2b02b0547c45d4bec86c9d7fadbfa2');
define('SECURE_AUTH_SALT', '92dcfcbb9076584b8181a8f7957e574e251f16cc27f66ca6fc71de7165ed4478');
define('LOGGED_IN_SALT',   '7f38f745b022cadab807dae933cad75a29794928f19cc025667d9fb72f5b4458');
define('NONCE_SALT',       'af61f686dc1eaeacb8df5c56d4548446fd5f127f07fdf014589bb60583000eb1');

/**#@-*/

/**
 * WordPress database table prefix.
 *
 * You can have multiple installations in one database if you give each
 * a unique prefix. Only numbers, letters, and underscores please!
 */
$table_prefix = 'wp_';

/**
 * For developers: WordPress debugging mode.
 *
 * Change this to true to enable the display of notices during development.
 * It is strongly recommended that plugin and theme developers use WP_DEBUG
 * in their development environments.
 *
 * For information on other constants that can be used for debugging,
 * visit the documentation.
 *
 * @link https://wordpress.org/support/article/debugging-in-wordpress/
 */
define( 'WP_DEBUG', false );

/* Add any custom values between this line and the "stop editing" line. */



/* That's all, stop editing! Happy publishing. */
/**
 * The WP_SITEURL and WP_HOME options are configured to access from any hostname or IP address.
 * If you want to access only from an specific domain, you can modify them. For example:
 *  define('WP_HOME','http://example.com');
 *  define('WP_SITEURL','http://example.com');
 *
*/

if ( defined( 'WP_CLI' ) ) {
    $_SERVER['HTTP_HOST'] = 'localhost';
}

define('WP_SITEURL', 'http://' . $_SERVER['HTTP_HOST'] . '/wordpress');
define('WP_HOME', 'http://' . $_SERVER['HTTP_HOST'] . '/wordpress');


/** Absolute path to the WordPress directory. */
if ( ! defined( 'ABSPATH' ) ) {
	define( 'ABSPATH', __DIR__ . '/' );
}

/** Sets up WordPress vars and included files. */
require_once ABSPATH . 'wp-settings.php';

define('WP_TEMP_DIR', 'C:\Users\Dell\Desktop\wp/apps/wordpress/tmp');


//  Disable pingback.ping xmlrpc method to prevent Wordpress from participating in DDoS attacks
//  More info at: https://docs.bitnami.com/general/apps/wordpress/troubleshooting/xmlrpc-and-pingback/

if ( !defined( 'WP_CLI' ) ) {
    // remove x-pingback HTTP header
    add_filter('wp_headers', function($headers) {
        unset($headers['X-Pingback']);
        return $headers;
    });
    // disable pingbacks
    add_filter( 'xmlrpc_methods', function( $methods ) {
            unset( $methods['pingback.ping'] );
            return $methods;
    });
    add_filter( 'auto_update_translation', '__return_false' );
}


apps/wordpress/htdocs/wp-content/uploads/themify_settings_themify-ultra_5.5.9_.php
<?php if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
$_arr=array (
  'setting-page_404' => '0',
  'setting-webfonts_list' => 'recommended',
  'setting-customizer_responsive_design_tablet_landscape' => '1024',
  'setting-customizer_responsive_design_tablet' => '768',
  'setting-customizer_responsive_design_mobile' => '600',
  'setting-mobile_menu_trigger_point' => '1100',
  'setting-color_design' => 'theme-color-black',
  'setting-header_design' => 'header-top-bar',
  'setting-exclude_search_form' => 'on',
  'setting_search_form' => 'search_form',
  'setting-exclude_header_widgets' => 'on',
  'setting-exclude_social_widget' => 'on',
  'setting-header_widgets' => 'headerwidget-3col',
  'setting-footer_design' => 'footer-left-col',
  'setting-use_float_back' => 'on',
  'setting-footer_widgets' => 'footerwidget-3col',
  'setting-mega_menu_posts' => '5',
  'setting-mega_menu_image_width' => '180',
  'setting-mega_menu_image_height' => '120',
  'setting-mega_menu_post_count' => 'off',
  'setting-color_animation_speed' => '5',
  'setting-relationship_taxonomy' => 'category',
  'setting-relationship_taxonomy_entries' => '3',
  'setting-relationship_taxonomy_display_content' => 'none',
  'setting-single_slider_autoplay' => 'off',
  'setting-single_slider_speed' => 'normal',
  'setting-single_slider_effect' => 'slide',
  'setting-single_slider_height' => 'auto',
  'setting-more_posts' => 'infinite',
  'setting-entries_nav' => 'numbered',
  'setting-footer_text_right_hide' => 'hide',
  'setting-gallery_lightbox' => 'lightbox',
  'setting-lazy-blur' => '25',
  'setting-cache-live' => '10080',
  'setting-webp-quality' => '5',
  'setting-default_layout' => 'sidebar1',
  'setting-default_post_layout' => 'list-post',
  'setting-post_filter' => 'no',
  'setting-disable_masonry' => 'yes',
  'setting-post_gutter' => 'gutter',
  'setting-default_layout_display' => 'content',
  'setting-default_more_text' => 'More',
  'setting-index_orderby' => 'date',
  'setting-index_order' => 'DESC',
  'setting-default_media_position' => 'above',
  'setting-image_post_feature_size' => 'blank',
  'setting-default_page_post_layout' => 'sidebar1',
  'setting-default_page_post_layout_type' => 'classic',
  'setting-default_page_single_media_position' => 'above',
  'setting-image_post_single_feature_size' => 'blank',
  'setting-search-result_layout_display' => 'excerpt',
  'setting-search-result_media_position' => 'above',
  'setting-default_page_layout' => 'sidebar-none',
  'setting-hide_page_title' => 'yes',
  'setting-comments_pages' => 'on',
  'setting-default_portfolio_index_layout' => 'sidebar-none',
  'setting-default_portfolio_index_post_layout' => 'grid3',
  'setting-portfolio_post_filter' => 'yes',
  'setting-portfolio_disable_masonry' => 'yes',
  'setting-portfolio_gutter' => 'gutter',
  'setting-default_portfolio_index_display' => 'none',
  'setting-default_portfolio_single_layout' => 'sidebar-none',
  'setting-default_portfolio_single_portfolio_layout_type' => 'fullwidth',
  'themify_portfolio_slug' => 'project',
  'themify_portfolio_category_slug' => 'portfolio-category',
  'setting-img_php_base_size' => 'large',
  'setting-global_feature_size' => 'blank',
  'setting-link_icon_type' => 'font-icon',
  'setting-link_type_themify-link-0' => 'image-icon',
  'setting-link_title_themify-link-0' => 'Twitter',
  'setting-link_img_themify-link-0' => 'http://127.0.0.1:81/wordpress/wp-content/themes/themify-ultra/themify/img/social/twitter.png',
  'setting-link_type_themify-link-1' => 'image-icon',
  'setting-link_title_themify-link-1' => 'Facebook',
  'setting-link_img_themify-link-1' => 'http://127.0.0.1:81/wordpress/wp-content/themes/themify-ultra/themify/img/social/facebook.png',
  'setting-link_type_themify-link-2' => 'image-icon',
  'setting-link_title_themify-link-2' => 'YouTube',
  'setting-link_img_themify-link-2' => 'http://127.0.0.1:81/wordpress/wp-content/themes/themify-ultra/themify/img/social/youtube.png',
  'setting-link_type_themify-link-3' => 'image-icon',
  'setting-link_title_themify-link-3' => 'Pinterest',
  'setting-link_img_themify-link-3' => 'http://127.0.0.1:81/wordpress/wp-content/themes/themify-ultra/themify/img/social/pinterest.png',
  'setting-link_type_themify-link-4' => 'font-icon',
  'setting-link_title_themify-link-4' => 'Twitter',
  'setting-link_ficon_themify-link-4' => 'fa-twitter',
  'setting-link_type_themify-link-5' => 'font-icon',
  'setting-link_title_themify-link-5' => 'Facebook',
  'setting-link_ficon_themify-link-5' => 'fa-facebook',
  'setting-link_type_themify-link-6' => 'font-icon',
  'setting-link_title_themify-link-6' => 'YouTube',
  'setting-link_ficon_themify-link-6' => 'fa-youtube',
  'setting-link_type_themify-link-7' => 'font-icon',
  'setting-link_title_themify-link-7' => 'Pinterest',
  'setting-link_ficon_themify-link-7' => 'fa-pinterest',
  'setting-link_field_ids' => '{"themify-link-0":"themify-link-0","themify-link-1":"themify-link-1","themify-link-2":"themify-link-2","themify-link-3":"themify-link-3","themify-link-4":"themify-link-4","themify-link-5":"themify-link-5","themify-link-6":"themify-link-6","themify-link-7":"themify-link-7"}',
  'setting-link_field_hash' => '8',
  'setting-twitter_settings_cache' => '10',
  'setting-recaptcha_version' => 'v2',
  'setting-page_builder_is_active' => 'enable',
  'setting-page_builder_gallery_lightbox' => 'enable',
  'setting-page_builder_animation_appearance' => 'none',
  'setting-page_builder_animation_parallax_bg' => 'none',
  'setting-page_builder_animation_scroll_effect' => 'none',
  'setting-page_builder_animation_sticky_scroll' => 'none',
);